src/Service/BattleEngine/BattlePhase.php

```diff
@@ -18,9 +21,6 @@ abstract class BattlePhase implements IBattlePhase
 
     protected string $name;
 
-    /**
-     * @var FleetUnit[]
```

Comment:
What is the reason you remove type hints in this MR? I use phpstorm and this tells the IDE its an array of FleetUnits, instead of just an array.

Reply:
Got confused due to the wrong typehint in Battlephase.